### PR TITLE
added pipeline.triggerType and trigger.type values

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 1]
```diff
@@ -493,6 +493,16 @@ The start time of a Pipeline execution in [Unix Epoch format](https://www.epoch1
 
 The type of Trigger. See [Trigger How-tos](/docs/category/triggers).
 
+Here are the possible `<+pipeline.triggerType>` and `<+trigger.type>` values.
+
+| **<+pipeline.triggerType>** | **<+trigger.type>** | **Description** |
+| :--- | :--- | :--- |
+| ARTIFACT | Artifact | New Artifact trigger. For example, new Docker Hub image tab |
```
Comment (konrness) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:500]: change: "new Docker Hub image tab" to "new Docker Hub image tag"

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 2]
```diff
@@ -493,6 +493,16 @@ The start time of a Pipeline execution in [Unix Epoch format](https://www.epoch1
...
+| SCHEDULER_CRON | Scheduled | Scheduled Cron trigger |
+| MANUAL | *null* | Pipeline triggered using the RUN button in the user interface |
+| WEBHOOK_CUSTOM | Custom | Custom webhook trigger |
+| WEBHOOK | Webhook | GitHub webhook trigger |
```
Comment (konrness) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:504]: The Webhook event can occur for all SCM's (e.g. Bitbucket), not just Github. Recommend changing description to:
SCM webhook trigger. For example, Github Pull Request.

Or something like that.

File: docs/platform/12_Variables-and-Expressions/harness-variables.md (hunk 1 above)
Comment (roshnisarangadharan) [reply docs/platform/12_Variables-and-Expressions/harness-variables.md]: Updated the entry to "new Docker Hub image tag". Check "Files changed" to see the changes.

File: docs/platform/12_Variables-and-Expressions/harness-variables.md (hunk 2 above)
Comment (roshnisarangadharan) [reply docs/platform/12_Variables-and-Expressions/harness-variables.md]: Updated the entry to "SCM webhook trigger. For example, GitHub pull request". Check "Files changed" to see the changes.

**Comment by roshnisarangadharan** [comment]
Closing this PR as the changes are taken care as part of https://github.com/harness/developer-hub/pull/643

**Pull request closed without merging**